assets/javascripts/discourse/components/query-result.js.es6

```diff
@@ -35,6 +35,16 @@ const QueryResultComponent = Ember.Component.extend({
     return this.get('content.columns').length;
   }.property('content.columns.length'),
 
+  resultCount: function() {
+    if (this.get('content.result_count') === 1) {
+      return I18n.t('explorer.single_result_count', {value: this.get('content.result_count')});
+    } else if (this.get('content.result_count') === 250) {
+      return I18n.t('explorer.result_limit_reached', {value: this.get('content.result_count')});
+    } else {
+      return I18n.t('explorer.total_result_count', {value: this.get('content.result_count')});
+    }
+  }.property('content.resultCount'),
```

Comment:
We should use computed decorator here. For reference see: https://github.com/discourse/discourse-push-notifications/blob/2f13c67a6128a2aae8548de2eeb15dd69bfc0618/assets/javascripts/discourse/components/push-notification-config.js.es6#L8-L11

Reply:
Done, thanks! Merging...

Reply:
wait

Reply:
@rishabhnambiar refer this code for correct implementation: https://github.com/discourse/discourse-canned-replies/blob/303f9525105b54c8553bb375fec91b07cb01cf2c/assets/javascripts/discourse/controllers/edit-reply.js.es6#L13-L17

Reply:
I hope it's correct now, thanks so much for showing me this pattern, it's pretty awesome! :100: